docker/Dockerfile.wifi-connect

```diff
@@ -0,0 +1,32 @@
+FROM balenalib/BALENA_ARCH-python:3.7-buster
```

Comment:
This should probably be refactored into a multi-stage build.

Reply:
Got it. For the second build stage, I can just use the base image that `screenly/srly-ose-base` is using, `balenalib/raspberypi3:buster`.